docs/src/contribute/design/next-tedge-api/topic-examples.md

```diff
@@ -0,0 +1,856 @@
+---
+title: Topic Examples
+tags: [Documentation]
+sidebar_position: 2
+---
+
+# Examples
+
+The following examples will demonstrate how the flexible topic structure can be used to provide rich modelling tools to publish data which is more observable to other components.
+
+## Example: Default device/service topic semantics
+
+### Publish to the main device
+
+```sh te2mqtt
+tedge mqtt pub -r 'te/device/main///m/environment' '{
+  "temperature": 23.4
+}'
+```
+
+If the there is no measurement type, then the type can be left empty, but it must have the trailing slash `/` (so that the number of topic segments is the same).
```

Comment:
What the meaning of this *measurement type* ? Is this the Cumulocity terminology or a kind of measurement such as temperature?

Reply:
The user is free to use *measurement_type* as the please. By using descriptive types (e.g. `realtime`, `1min_average`, or `environment_sensor`) it makes it possible for local mqtt client to subscribe to a subset of measurements.

But yes, Cumulocity IoT measurements support three different filtering possibilities,
type, valueFragmentSeries and valueFragmentType - [Reference](https://cumulocity.com/api/core/#operation/getMeasurementCollectionResource)